src/client/pages/matchPage/MatchPage.jsx

```diff
@@ -115,4 +140,8 @@ const MatchPage = () => {
     )
 }
 
+MatchPage.propTypes = {
```

Comment:
Just double checking. Would userId not be a number?

Reply:
It's a string in this case since it comes from localStorage. The backend handles the conversion to number :)